openbas-api/src/test/java/io/openbas/rest/InjectApiTest.java

```diff
@@ -39,6 +40,7 @@ class InjectApiTest extends IntegrationTest {
   static Document DOCUMENT2;
   static Team TEAM;
   static String SCENARIO_INJECT_ID;
+  static String EXERCISE_INJECT_ID;
```

Comment:
I don't think it's necessary to create a global variable since it's only used once. What do you think ?

Reply:
Since it is not used as it was anymore, it is better to delete it.